core/settings/base.py

```diff
@@ -144,5 +145,10 @@
 # Name of the URL where a user may login
 LOGIN_URL = 'auth_login'
 
-
 AUTH_USER_MODEL = 'register.User'
+
+OAUTH2_PROVIDER = {
```

Comment:
So I guess that's the setting we have missed during our pair session, kudos for finding it! 👍